build.gradle

```diff
@@ -219,14 +219,14 @@ repositories {
 }
 
 def versions = [
-  junit           : '5.11.3',
-  junitPlatform   : '1.11.3',
+  junit           : '5.7.2',
+  junitPlatform   : '1.7.2',
   reformLogging   : '5.1.9',
   springDoc       : '1.7.0',
   camunda         : '7.22.0',
-  pitest          : '1.17.1',
+  pitest          : '1.17.0',
   sonarPitest     : '0.5',
-  tomcat          : '11.0.0'
+  tomcat          : '10.1.31'
```

Comment:
Above gradle changes reverted as branch configs wasn't correct and renovate merged in and broke master